enterprise/core-edge/src/main/java/org/neo4j/coreedge/discovery/HazelcastClient.java

```diff
@@ -72,7 +72,7 @@ public ClusterTopology currentTopology()
         {
             log.info( "Failed to read cluster topology from Hazelcast. Continuing with empty (disconnected) topology. "
                     + "Connection will be reattempted on next polling attempt.", e );
-            return new ClusterTopology( false, emptyMap(), emptySet() );
+            return new ClusterTopology( null /* TODO */, false, emptyMap(), emptySet() );
```

Comment:
It is not very useful... no description of what to do and no hints about why we cannot Just Do It™